server/game/cards/06-WoE/SpaceInvaders.js

```diff
@@ -16,7 +16,8 @@ class SpaceInvaders extends Card {
                     target: context.player,
                     amount: context.target.length,
                     cards: context.target,
-                    cardLocation: 'hand'
+                    cardLocation: 'hand',
+                    alwaysSucceed: true
```

Comment:
I think something is wrong here.

Reply:
`MakeTokenCreature` checks that the deck is not empty, and that check isn't needed for this card.  `alwaysSucceed` was the easiest way to get around that but I'm open to other suggestions for sure.

Reply:
I believe it is the same root cause of https://github.com/keyteki/keyteki/issues/1840

Reply:
disregard the previous comment, I was thinking about TheVisibleHand.
This one probably needs a different GameAction. Let me think about it